std/format/write.d

```diff
@@ -20,76 +20,37 @@ import std.format.internal.write;
 import std.format.spec : FormatSpec;
 import std.traits : isSomeString;
 
-/**********************************************************************
-   Interprets variadic argument list `args`, formats them according
-   to `fmt`, and sends the resulting characters to `w`. The
-   encoding of the output is the same as `Char`. The type `Writer`
-   must satisfy $(D $(REF isOutputRange, std,range,primitives)!(Writer, Char)).
-
-   The variadic arguments are normally consumed in order. POSIX-style
-   $(HTTP opengroup.org/onlinepubs/009695399/functions/printf.html,
-   positional parameter syntax) is also supported. Each argument is
-   formatted into a sequence of chars according to the format
-   specification, and the characters are passed to `w`. As many
-   arguments as specified in the format string are consumed and
-   formatted. If there are fewer arguments than format specifiers, a
-   `FormatException` is thrown. If there are more remaining arguments
-   than needed by the format specification, they are ignored but only
-   if at least one argument was formatted.
-
-   The format string supports the formatting of array and nested array elements
-   via the grouping format specifiers $(B %&#40;) and $(B %&#41;). Each
-   matching pair of $(B %&#40;) and $(B %&#41;) corresponds with a single array
-   argument. The enclosed sub-format string is applied to individual array
-   elements.  The trailing portion of the sub-format string following the
-   conversion specifier for the array element is interpreted as the array
-   delimiter, and is therefore omitted following the last array element. The
-   $(B %|) specifier may be used to explicitly indicate the start of the
-   delimiter, so that the preceding portion of the string will be included
-   following the last array element.  (See below for explicit examples.)
-
```

Comment:
Isn't this information relevant? Why is it deleted?

Reply:
I thought I moved it to module scope. Must have made a mistake here.